Program/Seasons.py

```diff
@@ -0,0 +1,222 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures") 
+picfour = makePicture("Tree-Four-Seasons.jpg") 
```

Comment:
Please ensure that your variables names are following the [PEP-8 conventions](https://www.python.org/dev/peps/pep-0008/#function-names). Specifically the part which states: * names should be lowercase, with words separated by underscores as necessary to improve readability*